File: banda/JCZhang-Banda/JCZhang-Banda-JAVA/src/main/java/models/Singer.java
package models;

import interfaces.ISinger;

public class Singer extends Musician implements ISinger {

    String name;
    int yearsOfExperience;
    String tone;
    static double salary = (Musician.salary * 1.4);

    public Singer(String name, int yearsOfExperience, String tone) {
        super(name, yearsOfExperience);
        this.name = name;
        this.yearsOfExperience = yearsOfExperience;
        this.tone = tone;
    }

    @Override
    public void breath() {
        System.out.println("Im singer "+ name+ " and im breathing");
    }


    @Override
    public void sing() {
        System.out.println("Im a singer and im singing");
    }
}

File: banda/JCZhang-Banda/JCZhang-Banda_KOTLIN/src/main/kotlin/models/Singer.kt
package models

import interfaces.ISinger

class Singer(override var name: String?, override var yearsOfExperience: Int, var tone: String) :
    Musician(name, yearsOfExperience), ISinger {
    override fun breath() {
        println("Im singer $name and im breathing")
    }

    override fun sing() {
        println("Im a singer and im singing")
    }

    companion object {
        var salary = Musician.salary * 1.4
    }
}